etc/dnf/plugins/copr.conf

```diff
@@ -0,0 +1,6 @@
+[hub "fedora"]
+url = https://copr.fedorainfracloud.org
+
+# To add another Copr hub, make a new section with a URL to it.
+# [hub "name-of-hub"]
```

Comment:
What do you think to use directly hub name as section name like [fedora]? It will make code much easier. Additional what about to add section [main], where you can define default_hub=fedora? Or something like that.